app/src/main/res/raw/data.json

```diff
@@ -9699,23 +9702,1585 @@
     [
         {"EN":{
             "word": "cell",
-            "topics": ["science"],
+            "topics": ["science and technology"],
             "level": "upper-intermediate(B2)"
         }},
         {"RU": {
             "word": "клетка",
-            "topics": ["наука"]
+            "topics": ["наука и техника"]
         }}
     ],
     [
         {"EN":{
             "word": "carbohydrate",
-            "topics": ["science", "food"],
+            "topics": ["science and technology", "food"],
             "level": "upper-intermediate(B2)"
         }},
         {"RU": {
             "word": "углевод",
-            "topics": ["наука", "еда"]
+            "topics": ["наука и техника", "еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "light",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "свет",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "sun",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "солнце",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "computer",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "компьютер",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "menu",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "меню",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "mouse",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "мышь",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "open",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "открыть файл",
```

Comment:
I think translation is not correct, it means "open a file", not just "open". I would remove this word